addons/stock_dropshipping/models/stock.py

```diff
@@ -49,6 +49,26 @@ def _is_to_external_location(self):
         self.ensure_one()
         return super()._is_to_external_location() or self.is_dropship
 
+    def _send_confirmation_email(self):
+        subtype_id = self.env['ir.model.data']._xmlid_to_res_id('mail.mt_comment')
+        for stock_pick in self.filtered(lambda p: p.company_id.stock_move_email_validation and p.picking_type_id.code == 'dropship'):
+            delivery_template = stock_pick.company_id.stock_mail_confirmation_template_id
```

Comment:
super call will do there job don't need to call here standard template

Reply:
Hello @kir-odoo,

I don't think a super call will handle all the staff. I refer to the standard dropshipping template that doesn't have a purchase order as a source document because no template is being called in the super call for dropshipping.

Thank you!